### PR TITLE
gh-208. Add basic interface for an access-blob-storage package

### PR DESCRIPTION
**Pull request by yzhbankov**

### CODE REVIEW

**Comment by nesterone** [comment]
Regarding interface:
- blob storage should be generic for any blob content, in other words it's should not include any entities like Preview or CSV files 
- under 'generic' we mean that customers of it's API can write and read blobs, navigate over buckets/folders , generate links with auth access 

```js
class BlobStorage

    writeBlob(options, readableStream) {}
   
    makeFolder(options) {}
    
   / **
     * @returns {ReadableStream} 
    **/
    readBlob(options) {} 

    generateLink(options) {}
    
    list(folder, options)

    remove(options)

```


Interface is raw and for discussible, just to show an idea

File: packages/access-blob-storage/lib/minio/MinioPreviewStorage.mjs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  /**
+   * @param storage - Minio storage
+   */
+  constructor(storage) {
+    super()
+    this.#storage = storage
+  }
+
+  async save(params) {
+    return await this.#storage.putObject(this.#bucket, params.name, params.data)
```
Comment (nesterone) [inline packages/access-blob-storage/lib/minio/MinioPreviewStorage.mjs]: We should put some effort in 'wrapping' underling api of minio

for example 'just' returning original response from minio is not enought we should return our format of response, it
even became more interesting that we also need to take care for `errors`, so each call for underlined api should try to handle not only happy path but failure too

**Review by nesterone — CHANGES_REQUESTED** [review]

**Comment by yzhbankov** [comment]
@nesterone did you know a good example of a blob storage API? It is not clear to me what kind of interface should we implement and the use cases. Is the azure API(https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api, https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage, https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction#blob-storage-resources) could be an example?

File: packages/access-blob-storage/lib/minio/MinioStorage.mjs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+export class MinioStorage extends Storage {
+  /**
+   * @type {Minio}
+   */
+  #storage = null
+
+  constructor(options) {
+    super()
+    this.#storage = new Minio.Client({
```
Comment (nesterone) [inline packages/access-blob-storage/lib/minio/MinioStorage.mjs]: Configs should be taken from `.env`

**Comment by nesterone** [comment]
interface is ok, we need integration tests here to prof that module can work with minio

**Review by nesterone — CHANGES_REQUESTED** [review]

**Review by nesterone — APPROVED** [review]

**Pull request merged**